src/esn.calendar.libs/app/components/time-select/time-select.component.js

```diff
@@ -10,6 +10,6 @@ angular.module('esn.calendar.libs')
       onTimeChange: '=',
       locale: '@',
       timeFormat: '@',
-      disabled: '<?'
+      disabled: '=?'
```

Comment:
`?` character is redundant. Check angularjs component for more detail: https://docs.angularjs.org/guide/component